config/crd/bases/notification.kubesphere.io_notificationmanagers.yaml

```diff
@@ -828,6 +828,10 @@ spec:
                       type: object
                     global:
                       properties:
+                        cluster:
+                          description: The name of cluster which the notification
```

Comment:
The name of cluster which the notification manager be in.
=>
The name of the cluster in which the notification manager is deployed.